runtime/hydradx/src/lib.rs

```diff
@@ -450,19 +457,7 @@ use sp_core::OpaqueMetadata;
 use xcm_fee_payment_runtime_api::Error as XcmPaymentApiError;
 
 impl_runtime_apis! {
-	impl sp_api::Core<Block> for Runtime {
```

Comment:
nitpick: not sure what happened here but if possible, would you revert these changes

this PR should not touch/change anything else.